tests/tensor/test_sort.py

```diff
@@ -208,14 +209,16 @@ def test_argsort():
         utt.assert_allclose(gv, gt)
 
     # Example 5
-    a = dmatrix()
-    axis = lscalar()
-    a1 = ArgSortOp("mergesort", [])
-    a2 = ArgSortOp("quicksort", [])
+    a1 = ArgSortOp(
+        "mergesort",
+    )
+    a2 = ArgSortOp(
+        "quicksort",
+    )
     # All the below should give true
```

Comment:
Not your code, but this comment is extremely silly

Reply:
It is xD, I'll take it out